recipe/meta.yaml

```diff
@@ -21,6 +21,8 @@ build:
   number: {{ build_number }}
   skip: true  # [win and cuda_compiler != "None"]
   skip: true  # [linux and cuda_compiler == "None"]
+  # Skip Linux CUDA 12 ARM until RMM is available
+  skip: true  # [aarch64 and (cuda_compiler_version or "").startswith("12")]
```

Comment:
Skipping Linux CUDA 12 ARM for now until RMM is built for CUDA 12 ARM. Can drop this skip and re-render to add Linux CUDA 12 ARM builds

Reply:
Filed PR ( https://github.com/rapidsai/xgboost-feedstock/pull/29 ) to enable CUDA 12 ARM builds